pages/common/rustc.md

```diff
@@ -0,0 +1,19 @@
+# rustc
+
+> The Rust compiler.
+
+- Compile a single file:
+
+`rustc {{file.rs}}`
+
+- Compile with high optimization:
+
+`rustc -O {{file.rs}}`
+
+- Compile with debugging information:
+
+`rustc -g {{file.rs}}`
```

Comment:
Actually *rustc* takes the `-g` parameter as an alias of `-C debuginfo=2`  so the former is kind of a useful layer for ease of use, and there is no expanded option like `--debug` or anything like that. Should I use `-C debuginfo=2`  instead?